### ISSUE TITLE
Unsupported root element (ubl:CreditNote)

### ISSUE THREAD

**Issue by martinL70** [ZUGFeRD/mustangproject#428]
For a while now it has been mandatory in ubl to send an "invoice" with code 381 (credit note) as a "creditnote".
Mustang gives an error message ("Unsupported root element") for these.
Can this be included?

root-element:
<CreditNote xmlns="urn:oasis:names:specification:ubl:schema:xsd:CreditNote-2"...

**Comment by jstaerk** [comment]
It can be included, but in which context, ZUGFeRDInvoiceImporter?

**Comment by martinL70** [comment]
We need it in the validator.

**Comment by melo0187** [comment]
@jstaerk I would kindly ask to include the support for UBL CreditNote in the ZUGFeRDInvoiceImporter and ZUGFeRDValidator.

When I saw ae30a7a I was hoping this means there is already support for UBL CreditNote with 2.14.1 and newer in ZUGFeRDInvoiceImporter, because org.mustangproject.ZUGFeRD.ZUGFeRDImporter#getStandard returns EStandard, which was updated to contain `ubl_creditnote` in that commit.
But this method was never updated to deal with root element CreditNote.

Same is true for the ZUGFeRDValidator or more precisely XMLValidator, which has only implementation for root element Invoice but nothing for CreditNote.
